sdk/servicebus/azure_messaging_servicebus/README.md

```diff
@@ -0,0 +1,185 @@
+<!-- cspell:words pwsh yourgroup westus servicebus checkpointing  -->
+
+# Azure Service Bus client library for Rust
+
+Azure Service Bus is a fully managed enterprise message broker with message queues and publish-subscribe topics. Service Bus is used to decouple applications and services from each other, providing the following benefits:
+
+-   Load-balancing work across competing workers
+-   Safely routing and transferring data and control across service and application boundaries
+-   Coordinating transactional work that requires a high-degree of reliability
+
+[Source code] | [Package (crates.io)] | [API reference documentation] | [Product documentation]
+
+## WARNING - Not production ready crate
+
+This crate is in early development, it **SHOULD NOT** be used in production.
+
+## Getting started
+
+### Prerequisites
+
+-   Rust 1.85.0 or later
```

Comment:
Nit: don't document this. Harder to update and already apparent with our MSRV that is also surfaced elsewhere on some sites.